segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -307,8 +307,7 @@ public static void updateCoreAttributes(Map<Integer, String> backUpMetadataSegme
                                             ExecutorService executorService,
                                             Duration timeout) throws InterruptedException, ExecutionException,
             TimeoutException {
-        Preconditions.checkState(backUpMetadataSegments.size() == containersMap.size(), "The number of " +
-                "back-up metadata segments and containers should match.");
+        Preconditions.checkNotNull(backUpMetadataSegments, "Back up metadata segments");
```

Comment:
sometimes there are not all container metadata segments present in the LTS, so their back up copies didn't get created, and so the test failed. I think just the `notNull` check is sufficient.